docs/src/appendix/cast/multicall.md

```diff
@@ -27,4 +34,19 @@ call_type = "invoke"
 contract_address = "map_contract"
 function = "put"
 inputs = ["0x123", "234"]
-```
\ No newline at end of file
+```
+
+## `new`
+
+Generates an empty template for the multicall `.toml` file that may be later use with the `run` subcommand. It either outputs it to a new file or to the standard output.
+
+### `--output_path, -p <PATH>`
```

Comment:
```suggestion
### `--output-path, -p <PATH>`
```

Reply:
[underscore to dash](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/a89cfa5029992b32f6e23d35afa21d6ef1a7485f)